pkgs/development/compilers/eunomia-bpf/eunomia-cc/default.nix

```diff
@@ -0,0 +1,128 @@
+{ lib
+, stdenv
+, makeWrapper
+, fetchFromGitHub
+, rustPlatform
+, cmake
+, pkg-config
+, llvmPackages_16
+, cargo
+, rustc
+, elfutils
+, zlib
+, ...
+}:
+let
+  bpftool =
+    stdenv.mkDerivation {
+      pname = "bpftool";
+      version = "eunomia-edition-20230311";
+      src = fetchFromGitHub {
+        owner = "eunomia-bpf";
+        repo = "bpftool";
+        rev = "05940344f5db18d0cb1bc1c42e628f132bc93123";
+        hash = "sha256-g2gjixfuGwVnFlqCMGLWVPbtKOSpQI+vZwIZciXFPTc=";
+        fetchSubmodules = true;
+      };
+
+      buildInputs = [ llvmPackages_16.clang llvmPackages_16.libllvm.dev elfutils zlib ];
```

Comment:
So part of the purpose of the fork is the usage of clang libraries?

Reply:
> So part of the purpose of the fork is the usage of clang libraries?

There were some other modifications in fork https://github.com/eunomia-bpf/bpftool/compare/e1d17ae2570428d480c58ebaad296bf7591bb133...05940344f5db18d0cb1bc1c42e628f132bc93123